test/db/cmd/cmd_project

```diff
@@ -460,7 +460,7 @@ NAME=Save project for a file with x86_16 and check the value after loading again
 FILE=bins/src/hello.c
 ARGS=
 CMDS=<<EOF
-e prj.git=false
+e prj.vc=false
```

Comment:
what about having prj.vc.type=git ? to select the vc to use for projects?

Reply:
How can I do this ?